esmvalcore/cmor/table.py

```diff
@@ -29,6 +29,41 @@
 CMOR_TABLES: dict[str, CMORTable] = {}
 """dict of str, obj: CMOR info objects."""
 
+_CMOR_KEYS = (
+    'standard_name',
+    'long_name',
+    'units',
+    'modeling_realm',
+    'frequency',
+)
+
+
+def _update_cmor_facets(facets, override=False):
+    """Update `facets` with information from CMOR table."""
+    project = facets['project']
+    mip = facets['mip']
+    short_name = facets['short_name']
+    derive = facets.get('derive', False)
+    table = CMOR_TABLES.get(project)
+    if table:
+        table_entry = table.get_variable(mip, short_name, derive)
+    else:
+        table_entry = None
+    if table_entry is None:
```

Comment:
```suggestion
```

Reply:
I would be hesitant to remove that because it is probably in the current code for a reason:
https://github.com/ESMValGroup/ESMValCore/blob/155335a4b16ea911f546f9d6b4909bd427ae0af2/esmvalcore/_recipe.py#L94-L97

Reply:
ha OK I was looking about too, couldn't find any breakage point but heyho, not gonna improve complexity if we remove one single line, but that might break stuff downstream :laughing: